src/Nordea/Components/DropdownFilter.elm

```diff
@@ -99,9 +57,14 @@ type alias DropdownFilterProperties a msg =
     , hasError : Bool
     , isLoading : Bool
     , hasSearchIcon : Bool
-    , selectedValue : Maybe a
+    , selectedValue : Maybe (Item a)
```

Comment:
This change is breaking, but I have the PRs ready for FF, PH. I can do KP too. It is an easy change.

Reply:
Do we need this change? I guess we could get the `item` from the `a` by searching through the `items` list. That way, we can avoid saving the translation text in the [model in PH](https://github.com/SGFinansAS/PartnerHub/pull/2689/files#diff-464277551852faee39e9bae9bfc824bee95be95fe70b59fd9fa928e83aa4d9c5R25). Not a big deal tho

Reply:
Hmm, true. I might do that search here and see how it will look.

Reply:
Alright so keep it backwards compatible I applied this change but in general interface feels a bit awkward because of the mismatch between msg and stored value. They should be either `Maybe a` or `Maybe (Item a)`  together

```
init :
    { onInput : String -> msg
    , input : String
    , onSelect : Maybe (Item a) -> msg
    , selectedValue : Maybe a
    , items : List (ItemGroup a)
    }
```

Reply:
Ah yes I agree. I vote for `Maybe a`